src/main/java/pl/javastart/movieclub/domain/comment/Comment.java

```diff
@@ -0,0 +1,41 @@
+package pl.javastart.movieclub.domain.comment;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import pl.javastart.movieclub.domain.movie.Movie;
+import pl.javastart.movieclub.domain.user.User;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import java.time.LocalDateTime;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "movie_comment")
+public class Comment {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long Id;
+
+    @ManyToOne()
```

Comment:
nie powinno to byc jesszcze oznaczone adnotacją https://github.com/NotNull? (tez zalezne od tego czy dodales javax validations)